ros_ws/src/car_control/include/car_control.h

```diff
@@ -4,15 +4,15 @@
 
 #include <time.h>
 
+#include <drive_msgs/drive_param.h>
 #include <std_msgs/Float64.h>
-#include <std_msgs/Int64.h>
 #include <std_msgs/String.h>
 
 #define TOPIC_FOCBOX_SPEED "/commands/motor/speed"
 #define TOPIC_FOCBOX_ANGLE "/commands/servo/position"
 
-#define TOPIC_SPEED "/set/speed"
-#define TOPIC_ANGLE "/set/angle"
+#define TOPIC_DRIVE_PARAM "/set/drive_param"
+#define TOPIC_CMD "/cc/cmd"
```

Comment:
Please rename the topic so that the name explains what this topic does. Please also rename the directive.

Reply:
Please also rename the directive.

Explain

Reply:
Please also rename the directive.

Explain

Reply:
The part that says `TOPIC_CMD` should say something else.

Reply:
done